schemas/profile-functions.v1.schema.json

```diff
@@ -41,21 +35,6 @@
         },
         "transaction_name": {
           "type": "string"
-        },
-        "transaction_op": {
-          "type": "string"
-        },
-        "transaction_status": {
-          "type": "string"
-        },
-        "http_method": {
-          "type": "string"
-        },
-        "browser_name": {
-          "type": "string"
-        },
-        "device_class": {
-          "type": "integer"
```

Comment:
As discussed, these fields were never exposed to the user and no users have asked for them, so let's go ahead and remove them.